src/main/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinter.java

```diff
@@ -172,34 +168,4 @@ private static DetailNode parseFile(File file) throws IOException {
             System.getProperty("file.encoding", "UTF-8"));
         return parseJavadocAsDetailNode(text.getFullText().toString());
     }
-
-    /**
-     * Creates DetailAST block comment to pass it to the Javadoc parser.
-     * @param content comment content.
-     * @return DetailAST block comment
-     */
-    private static DetailAST createFakeBlockComment(String content) {
```

Comment:
Looking at comment made by @rnveach at [4643#discussion](https://github.com/checkstyle/checkstyle/pull/4643#discussion_r128122810), it seeems that he might want to revert this change. @rnveach Can you please address this point too. Thanks.